integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/ppl/StatsCommandIT.java

```diff
@@ -81,4 +80,22 @@ public void testStatsCount() throws IOException {
   }
 
   // TODO: each stats aggregate function should be tested here when implemented
+
+  @Test
+  public void testStatsNested() throws IOException {
+    String result =
+        executeQueryToString(
+            String.format("source=%s | stats avg(abs(age)*2) as AGE", TEST_INDEX_ACCOUNT));
+    assertEquals(
```

Comment:
when should we use plan text comparsion? when shoud with use matchers like verifyColumn?

Reply:
To my understanding, we can use verifyColumn like matchers in most of time, but some cases that some other content other than columns and datarows are also important to veriy, so we can flexibly choose text comparison. For example, to verify the formats, or verify the explain results etc. 
As for the case this comment referred to, I would like to verify schema, column names, result set of the aggregation, so I chose to compare text.